src/tools/nb.js

```diff
@@ -41,6 +41,7 @@ nb.sameKeys = function(x,y){
 
 nb.isRowDf = function(obj,rowsToCheck = 100){
   if (!_.isArray(obj) || obj.length==0) {return false}
+  if (obj.length === 1 && !_.isPlainObject(obj[0])) {return false}
```

Comment:
Would probably mash this into the if statement above, since they return false. i.e.

```js
if (!_.isArray(obj) || obj.length==0 || (obj.length === 1 && !_.isPlainObject(obj[0])) {
  return false;
}
```

Also, some tests would be nice.